app/components/collections/form_component.html.erb

```diff
@@ -5,22 +5,22 @@
     <header>Details</header>
     <div class="mb-3 row">
       <%= form.label :name, 'Collection name', class: 'col-sm-2 col-form-label' %>
-      <div class="col-sm-10">
+      <div class="col-sm-8">
```

Comment:
Are you sure about `sm`?  That begins at 540px.  How about leaving `col-sm-10` and adding `col-xl-8`?

Reply:
@jcoyne I have no idea. I'm just tweaking what's already there. Re: what column sizing we already have in the app: there are currently 0 `col-xl-*`, 4 `col-lg-*`, and 55 'col-sm-*`. I'm happy to make changes but feel a bit like I'm flying blind.

Reply:
@jcoyne happy to pair on this tomorrow.

Reply:
@jcoyne done. take another look?